releasetasks/templates/fennec/candidates_fennec.yml

```diff
@@ -0,0 +1,57 @@
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    reruns: 5
+    task:
+        provisionerId: aws-provisioner-v1
+        workerType: gecko-decision
+        created: "{{ now }}"
+        deadline: "{{ now.replace(days=4) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        routes: []
+        scopes:
+             - docker-worker:cache:level-3-*
+             - docker-worker:cache:tooltool-cache
+             - scheduler:extend-task-graph:*
+             - assume:hook-id:project-releng/candidates-fennec-dev,
```

Comment:
We may want to create a new role for fennec releases, rather than depend on candidates-fennec-dev.  This can be a followup fix if we want to test as-is at first.